Sources/Clappr/Classes/Base/Core.swift

```diff
@@ -92,6 +93,24 @@ open class Core: UIObject, UIGestureRecognizerDelegate {
             activeContainer = container
         }
     }
+    
+    public func updateChromelessMode(with option: Options) {
```

Comment:
I have a few considerations about this method.

I don't think it should be public. It kinda allows for an external caller to pass a different set of `Options` to `Core`, other than the ones received on the initializer or directly modified using the `core.options` stored property. 

Also, it seems to me that it is called in the wrong places. It seems to me that the actual trigger to this change is the modification of the `core.options` property. As we already have a `didSet` callback for it, we could call it there or better yet, **inline it**.